src/core/lib/security/credentials/plugin/plugin_credentials.cc

```diff
@@ -42,6 +42,16 @@ grpc_plugin_credentials::~grpc_plugin_credentials() {
   }
 }
 
+std::string grpc_plugin_credentials::debug_string() const {
+  if (plugin_.state != nullptr) {
+    char* debug_c_str = plugin_.debug_string(plugin_.state);
```

Comment:
Does this need to check if `plugin_.debug_string` is `nullptr` (for plugins that don't implement `debug_string`)?

Reply:
Thanks for the pointer. Fixed